source/kubernetes.rst

```diff
@@ -1,44 +1,46 @@
-Install Percona XtraDB Cluster on Kubernetes
+.. _kubernetes:
+
+Install the |Percona Operator PXC| on Kubernetes
 ============================================
 
 0. First of all, clone the percona-xtradb-cluster-operator repository:
 
-   .. code:: bash
+   .. code-block:: bash
 
-      git clone -b release-0.3.0 https://github.com/percona/percona-xtradb-cluster-operator
+      git clone -b release-1.0.0 https://github.com/percona/percona-xtradb-cluster-operator
       cd percona-xtradb-cluster-operator
 
-   **Note:** *It is crucial to specify the right branch with ``-b``
-   option while cloning the code on this step. Please be careful.*
+   **Note:** *You must specify the correct branch with ``-b``
```

Comment:
it is really crucial, it is the most popular bug, so the user should be careful here.